[docs] Adoption of existing Node under Cluster API Provider Static (#9282)
deckhouse/deckhouse: modules/040-node-manager/docs/README.md

   - `Running`. The server is configured and the associated node is added to the cluster.
   - `Cleaning`. The procedure of cleaning up the server and disconnecting the node from the cluster is in progress.

>    > It is possible to transfer an existing manually-bootstrapped cluster node under CAPS management by annotating its StaticInstance as `static.node.deckhouse.io/skip-bootstrap-phase: ""`.

Reviewer: ```suggestion
   > You can transfer the existing manually-bootstrapped cluster node under CAPS management by annotating its StaticInstance with `static.node.deckhouse.io/skip-bootstrap-phase: ""`.
```
